bindings/python/capstone/__init__.py

```diff
@@ -110,6 +110,8 @@
     'CS_OPT_ON',
     'CS_OPT_OFF',
 
+    'CS_OPT_MEM',
```

Comment:
Shouldn't it be `CS_OP_MEM`?

Reply:
Just saw that this has been fixed in your autosync branch